### PR TITLE
[MSHARED-1113] Downgrade m-resource-p to 3.1.0

### PR DESCRIPTION
**Pull request by slawekjaranowski**
Cause MSHARED-1112

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -119,6 +119,17 @@
   </dependencies>
 
   <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <!-- review with next pom or next m-resources-p -->
```
Comment (kwin) [inline pom.xml:125]: Please explicitly mention https://issues.apache.org/jira/browse/MSHARED-1113 to ease reviewing

Comment (kwin) [inline pom.xml:125]:
```suggestion
         <!-- downgrade due to https://issues.apache.org/jira/browse/MSHARED-1112, review with next pom or next m-resources-p -->
```

**Review by kwin — APPROVED** [review]
LGTM, thanks.

**Comment by jira-importer** [comment]
Resolve #309

**Pull request merged**